docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -283,6 +283,9 @@ Database resource group manager
 
 This example is for a MySQL database.
 
+As stated above, the environment ``'test'`` in the example should match ``node.environment`` property from
```

Comment:
Why would we repeat this here when it is clearly stated earlier .. by that logic we should explain all other parameters again as well. ... if anything you could change the value to be more explicit something like `test_environment` or whatever..